src/test/java/guitests/UITest.java

```diff
@@ -337,4 +337,11 @@ public GuiTest type(String text) {
         return this;
     }
 
+    /**
+     * Used to select the whole filter text so that it can be replaced
+     */
+    public void selectAll() {
+        pushKeys(KeyCode.END);
```

Comment:
I added press end to make it usable for any purpose, in case the cursor is
not at the end of text box. Should I remove it? Seems safer to me.
On Nov 10, 2015 1:17 PM, "Darius Foo" <notifications@github.com> wrote:

> In src/test/java/guitests/UITest.java
> <https://github.com/HubTurbo/HubTurbo/pull/1123#discussion_r44371106>:
>
> > @@ -337,4 +337,11 @@ public GuiTest type(String text) {
> >          return this;
> >      }
> >
> > +    /**
> > +     * Used to select the whole filter text so that it can be replaced
> > +     */
> > +    public void selectAll() {
> > +        pushKeys(KeyCode.END);
>
> Do we still need the End key press?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/HubTurbo/HubTurbo/pull/1123/files#r44371106>.
>